std/js/Lib.hx

```diff
@@ -71,6 +71,17 @@ class Lib {
 		return untyped __js__("undefined");
 	}
 
+	/**
+		Checks a value against JavaScript `undefined` value.
+
+		Note that this is only needed in very rare cases when working with external JavaScript code.
+
+		In Haxe, `null` is used to represent the absence of a value.
+	**/
+	public static inline function isUndefined<T>(value:T):Bool {
+		return untyped __js__("{0} === undefined", value);
```

Comment:
Please use `Syntax.strictEq(value, undefined)`.

Maybe we should also change `get_undefined` to return `Syntax.code("undefined")` to reduce the `untyped`ness.